src/components/books/AddnewBook.module.css

```diff
@@ -2,6 +2,11 @@
   color: #c7c7c7;
 }
 
+.error {
+  color: red;
+  font-size: 10px;
+}
+
```

Comment:
- [ ] Please align the input boxes in one line as the original design.

Your design 

![Capture](https://user-images.githubusercontent.com/64173189/150618094-3eb97f79-5a4e-4f10-a191-34b5d371cee0.JPG)

Original design

![Capture](https://user-images.githubusercontent.com/64173189/150618251-5495c80d-cca3-4dbb-9ce8-70df3b1ea1c3.JPG)